website/docs/reference/widgets/list.md

```diff
@@ -249,34 +130,16 @@ You can do some formatting changes to enhance the look and feel of the widget by
 | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Change the heading of the column to just 'Property' instead of  'Style property'